kinto/core/utils.py

```diff
@@ -423,3 +423,24 @@ def instance_uri(request, resource_name, **params):
     """Return the URI for the given resource."""
     return strip_uri_prefix(request.route_path('%s-record' % resource_name,
                                                **params))
+
+
+def parse_resource(resource):
+    """extract the bucket and collection of a given resource"""
```

Comment:
* Nitpick: capital `Extract`
* What is resource here? → It's a URI
* What are bucket and collection → they are bucket id and collection id
* What does the function return? A tuple? A dict ?